config/packages/framework.yaml

```diff
@@ -17,7 +17,7 @@ framework:
     http_client:
         default_options:
             headers:
-                'User-Agent': 'MbinBot/0.1 (+https://%kbin_domain%/bot)'
+                'User-Agent': 'Mbin/0.1 (+https://%kbin_domain%)'
```

Comment:
Ooo, I see this still has the old version number in it... probably should update it to match what's in `ProjectInfoService.php`? We might have to rework how we're handling a global var to change the version everywhere it's called... could be forced to put it the `.yaml` unless there's a way to call it in from PHP somewhere.